packages/wallet/backend/src/utils/helpers.ts

```diff
@@ -1,5 +1,32 @@
 import { bcrypt } from 'hash-wasm'
 import { randomBytes } from 'crypto'
+import { ObjectWithAnyKeys } from '@/shared/types'
+
+export const replaceObjectKey = (
```

Comment:
is this used somewhere?

Reply:
Not at the moment. We will keep this for the moment since we had some issues with deep relations filters.